time stamp
string save_path
---
string save_path
string bag_name
bool is_running
